fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.json

```diff
@@ -294,21 +294,25 @@
    "print": 1,
    "read": 1,
    "report": 1,
-   "role": "System Manager",
+   "role": "Administrator",
    "share": 1,
    "write": 1
   },
   {
-   "create": 1,
+   "delete": 1,
    "email": 1,
    "export": 1,
-   "print": 1,
    "read": 1,
    "report": 1,
-   "role": "All",
-   "select": 1,
-   "share": 1,
-   "write": 1
+   "role": "System Manager"
```

Comment:
is System Manager > Administrator or Administrator > System Manager?

Reply:
Administrator > System Manager. 

We can only log in as Administrator through Frappecloud. Administrator is the Super user.

Reply:
> We can only log in as Administrator through Frappecloud.

and taking over a foss united profile does not give them access to frappecloud because the authentication for that happens separately. correct?

Reply:
Does the system manager need all of these roles? Should they only be able to read and delete?

Reply:
Administrator > System Manager

Reply:
Gave the permission to email because Ruchika and I often work on ticket corrections and send corrected tickets/invoices through desk, which needs the permission for emails.

export can be removed.

Report is in case of insights, as we are doing a lot of workspaces in desk.

Reply:
> Gave the permission to email because Ruchika and I often work on ticket corrections and send corrected tickets/invoices through desk, which needs the permission for emails.

thanks for the explainer. there are User Personas emerging from this discussion that can be used in the future to refine permissions better.